content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -13,23 +13,56 @@ This is a small example application we will monitor in this guide.
 
 Create an empty package.json:
 
-```sh
+{{< tabs TypeScript JavaScript >}}
+
+{{< tab >}}
+tsc --init
+{{< /tab >}}
+
+{{< tab >}}
 npm init -f
-```
+{{< /tab >}}
+
+{{< /tabs >}}
 
 Install dependencies used by the example.
 
-```sh
+{{< tabs TypeScript JavaScript >}}
+
+{{< tab >}}
+npm install express typescript ts-node express @types/express @types/node nodemon
+{{< /tab >}}
+
+{{< tab >}}
 npm install express
-```
+{{< /tab >}}
+
+{{< /tabs >}}
 
 ### Code
 
-Please save the following code as `app.js`.
+Please save the following code as `app.ts` or `app.js`.
```

Comment:
```suggestion
Save the following code as `app.ts` or `app.js`, as appropriate:
```

Reply:
I have added `add.ts|js` like earlier suggested for `document-load.ts|js`